packages/orchestration/src/service.js

```diff
@@ -96,26 +96,26 @@ const prepareChainAccount = zone =>
        *   localAddress: string | undefined;
        *   requestedRemoteAddress: string;
        *   remoteAddress: string | undefined;
-       *   accountAddress: string | undefined;
+       *   address: ChainAddress['address'] | undefined;
        * }}
        */ (
         harden({
           port,
           connection: undefined,
           requestedRemoteAddress,
           remoteAddress: undefined,
-          accountAddress: undefined,
+          address: undefined,
```

Comment:
should this be "localAddress" in comparison to remote address?  or is it "sourceAddress"?

Reply:
The localAddress means something different - it's the channel address on the local chain. Here is console output from a test:
```
 ICA Account Addresses {
        accountAddress: 'cosmos1test',
        localAddress: '/ibc-port/icacontroller-1/ordered/{"version":"ics27-1","controllerConnectionId":"connection-0","hostConnectionId":"connection-0","address":"cosmos1test","encoding":"proto3","txType":"sdk_multi_msg"}/ibc-channel/channel-1',
        remoteAddress: '/ibc-hop/connection-0/ibc-port/icahost/ordered/{"version":"ics27-1","controllerConnectionId":"connection-0","hostConnectionId":"connection-0","address":"cosmos1test","encoding":"proto3","txType":"sdk_multi_msg"}/ibc-channel/channel-NaN',
      }
```

We need to extract `address` from the negotiated remote channel address.